### PR TITLE
(MODULES-3641) Remove Windows_env module dependency

### PR DESCRIPTION
**Pull request by ferventcoder**
It was originally required to set the environment variables prior
to running Chocolatey install to help prepare the environment for
suitability. It was originally added in 5a849adca and ad57d1429
to ensure that Chocolatey could be suitable to be installed and
used to install packages on the first run. This was also to
ensure that you could override the Chocolatey install directory
by setting the value in advance of the install.

In b5034d167 @rismoney removed the necessity for pre-setting the
environment variable as it was passed to the install exec.

We have since determined that the providers default to a location
of `$env:ALLUSERSPROFILE\chocolatey\bin\choco.exe` when an install
path cannot be found. https://github.com/puppetlabs/puppetlabs-chocolatey/blob/fa67417bbfac06049f36346e02eb09dddb68084c/lib/puppet_x/chocolatey/chocolatey_common.rb#L31-L36
This evaluates to the default install path of
`C:\ProgramData\chocolatey\bin\choco.exe`, allowing suitability
to work properly on the same converge as the install ONLY if you
install to the default install location with Chocolatey. This
has nothing to do with the hope that windows_env would update
the current shell! So it is safe to remove windows_env without
losing any of the current behavior.

### CODE REVIEW

**Comment by ferventcoder** [comment]
Rebased against #33

**Comment by ferventcoder** [comment]
@puppetlabs/windows please review.

File: .fixtures.yml [hunk 1]
```diff
@@ -2,6 +2,5 @@ fixtures:
   forge_modules:
     stdlib:       "puppetlabs/stdlib"
     powershell:   "puppetlabs/powershell"
-    windows_env:  "badgerious/windows_env"
```
Comment (ferventcoder) [inline .fixtures.yml]: @jamesp noticed this was still in here.

**Comment by jpogran** [comment]
Found MODULES-3677 during testing, work to fix that requires new ticket and is not related to this one. 👍

**Pull request merged**